### DISCUSSION TITLE
Title suggestions

### DISCUSSION THREAD

**Discussion by t3chn0T33n-zz** [fnuecke/oc2#34] [Ideas]
First off, I understand that the title is currently low priority, so no need to focus on this discussion.

Ok, bullet point time!

- RISCy Computers
- RISCed Computers
- Linux Computers
- R.O.M.C (RISC Operating Minecraft Computers)

**Comment by Faeranne** [comment]
Honestly, OC2 or Open Computers 2 seems fine to me.  But I can get wanting to differentiate due to the underlying architecture change.

Something easy to say seems better, but Linux Computers is too... generic?  Idk.  Just my thoughts.  Names are hard.

**Reply by t3chn0T33n-zz** [reply]
I was kinda just listing to see if one catches an eye. But I do agree "linux Computers" is kinda bland...

**Reply by griffi-gh** [reply]
the name "opencomputers 2" is kinda bad.  
the mod is almost entirely different, so the name should be too

**Reply by t3chn0T33n-zz** [reply]
Exactly my thoughts.

**Comment by t3chn0T33n-zz** [comment]
I've thought of more:

- M.O.C.I.M (Modular Operating system Computers In Minecraft) 2/5

- Modular Computers 5/5

- M.C.M (Modular Computers in Minecraft) 4/5

- M.O.M (Minecraft Open Modular) "Did you get the Mom mod?" Meme/5

**Reply by CoolCat467** [reply]
Open Modular Computers?
Better Open Computers?

**Reply by Faeranne** [reply]
Another vote for Open Modular Computers.  But I think this is really up to Sangar.  He might just stick with OpenComputers.

**Reply by CoolCat467** [reply]
Yea, probably right

**Reply by t3chn0T33n-zz** [reply]
"Open Modular Computers" Yes, it keeps a bit of the original name but also pushes the fact that it's different in functionality! As a certain tall comedian said, "Very nice!"

**Comment by Emmixis** [comment]
I do enjoy something with the word "Modular". I feel having "RISC" in the title may scare off new users, where as something short and sweet like "Modular Computers" could be easily appealing. Just my opinion, though, glad to hear other people's thoughts.

**Reply by CoolCat467** [reply]
I agree about the RISC bit

**Comment by atirutw** [comment]
So OC2 uses RISC architecture for it's VM?

**Reply by CoolCat467** [reply]
I think so

**Reply by t3chn0T33n-zz** [reply]
Yes.

**Reply by Klebestreifen** [reply]
RISC-V

**Comment by CitrusSin** [comment]
How about "Modular Simulated RISC Machines(M.S.R.M.)"?

**Reply by CitrusSin** [reply]
This mod is not only about computers but also robots, drones, and microcontrollers. So I think "computers" might be a little limited, and I use "RISC Machines" instead

**Reply by CoolCat467** [reply]
That might be good

**Comment by patryk3211** [comment]
What about "Computers-V" since this mod emulates the RISC-V architecture.

**Comment by griffi-gh** [comment]
Computech?

**Reply by atirutw** [reply]
I think this one is the best out of all suggestions as of currently.

**Comment by Klebestreifen** [comment]
Compunk

**Comment by rsp4jack** [comment]
The Minecraftic open virtual modular RISC-V computers based Linux
MOVMCL
(Just a joke...)

**Comment by fnuecke** [comment]
Thanks for all the suggestions. Will OpenComputers II will stick after all, though. Among other things, too lazy to change the naming everywhere now (somewhat predictably, in hindsight, oh well).

**Comment by repo-alt** [comment]
Eh, this is so different from the previous OpenComputers...

**Comment by griffi-gh** [comment]
NeoComputers, maybe?
not too different from OpenComputers
